beckFilter/challenges-08.test.js

```diff
@@ -0,0 +1,309 @@
+'use strict';
```

Comment:
You've got your `filter` solutions in the chaining methods branch. Make sure you're checking out master before you check out the new branch to start each day.